starboard/player.h

```diff
@@ -450,11 +452,12 @@ SbPlayerGetPreferredOutputMode(const SbPlayerCreationParam* creation_param);
 // Destroys |player|, freeing all associated resources.
 //  * Upon calling this method, there should be one call to the player status
 //    callback (i.e. |player_status_func| used in the creation of the player)
-//    which indicates the player is destroyed. Note, the callback has to be
-//    in-flight when SbPlayerDestroyed is called.
+//    which indicates the player is destroyed. Note, the callback has to
+//    be in-flight when SbPlayerDestroyed is called.
```

Comment:
"be" should be in the previous line.  Generally we fit line to as close to 80 characters between breaking them.

Reply:
Changed `in-flight` to `inflight`, as it mis-generated a space between in and -flight on our docsite.